lib/rules/object-curly-spacing.js

```diff
@@ -121,9 +121,10 @@ module.exports = function(context) {
      * @returns {void}
      */
     function validateBraceSpacing(node, first, second, penultimate, last) {
+        var penultimateType = sourceCode.getNodeByRangeIndex(penultimate.start).type;
         var closingCurlyBraceMustBeSpaced =
```

Comment:
If this is expensive, `closingCurlyBraceMustBeSpaced ` calculation could be moved [here](https://github.com/eslint/eslint/pull/5545/files#diff-9da030a8c7c9a4eea24f42982a0391b0L141)